charts/spire/charts/spiffe-oidc-discovery-provider/README.md

```diff
@@ -26,8 +26,7 @@ A Helm chart to install the SPIFFE OIDC discovery provider.
 | config.acme.directoryUrl | string | `"https://acme-v02.api.letsencrypt.org/directory"` |  |
 | config.acme.emailAddress | string | `"letsencrypt@example.org"` |  |
 | config.acme.tosAccepted | bool | `false` |  |
-| config.domains[0] | string | `"localhost"` |  |
-| config.domains[1] | string | `"oidc-discovery.example.org"` |  |
+| config.domains | list | `[]` | Override the domains to use. Defaults to jwtIssuer and localhost. |
```

Comment:
I would still prefer to have those defaults as real defaults in values.yaml.

Reply:
I'm trying to simplify what a user would have to do to deploy a production system.

```
global:
  jwtIssuer: thingy
```

is much easier then:
```
spire-server:
  jwtIssuer: thingy
spiffe-oidc-provider:
  domains:
  - localhost
  - thingy
```

---

The distinct(concat... thing could help though. What if we set config.domain to be just ['local'] by default and have the rest of the patch as is? Might meet both needs a little better?

Reply:
Added the uniq(concat thing.